client/src/pages/Subject/Subject.css

```diff
@@ -0,0 +1,32 @@
+.page-content {
+	padding: 10%;
+}

```

Comment:
Add this to a separate class. Since we use `.page-content` on almost every page, this affects every single page!